homeassistant/components/rako/light.py

```diff
@@ -0,0 +1,202 @@
+"""Platform for light integration."""
+import asyncio
+import logging
+from typing import Any, Callable, Dict, List, Optional
+
+import python_rako
+from python_rako.exceptions import RakoBridgeError
+from python_rako.helpers import convert_to_brightness, convert_to_scene
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS,
+    LightEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import HomeAssistantType
+
+from . import DATA_RAKO_BRIDGE_CLIENT, DOMAIN
+from .const import ATTR_CHANNEL_ID, ATTR_MANUFACTURER, ATTR_ROOM_ID
+from .util import create_unique_id
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType,
```

Comment:
"This is a cleaner approach and makes click-though actually useful."
`from homeassistant.core import HomeAssistant`
```suggestion
    hass: HomeAssistant,
```